mkdocs.yml

```diff
@@ -11,6 +11,7 @@ nav:
     - Installing rAthena:
       - General: installing.md
       - Installing on Debian: installing/debian.md
+      - Running with Docker: installing/docker.md
```

Comment:
Would "Using Docker" make more sense in this instance? I'm undecided if I "Running with" is the correct term.

Reply:
I like the suggested template too.

Reply:
I agree, that looks good.

Reply:
Done in [f888508](https://github.com/rathena/user-guides/pull/2/commits/f8885088a0614a9491703f7c5467a15c0fcdc278) 👍🏼